tests/Integration/Tickets/Ignored/IndexTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Aviator\Helpdesk\Tests\Integration\Tickets\Ignored;
+
+use Carbon\Carbon;
+use Aviator\Helpdesk\Tests\TestCase;
+
+class IndexTest extends TestCase
+{
+    /** @var string */
+    protected $url = 'helpdesk/tickets/ignored';
+
+    /** @test */
+    public function guests_are_redirected_to_login ()
+    {
+        $response = $this->get($this->url);
+
+        $response->assertStatus(302)
+            ->assertRedirect('login');
+    }
+
+    /** @test */
+    public function only_supers_see_ignored_tickets ()
+    {
+        $this->withoutErrorHandling();
+        $user = $this->make->user;
+        $agent = $this->make->agent;
+        $ignoredUser = $this->make->user;
+        $super = $this->make->super;
+
+        $this->addIgnoredUser([$ignoredUser->email]);
+
+        $ticket1 = $this->make->ticket($user);
+        $ticket2 = $this->make->ticket($ignoredUser);
+
+        $response = $this->actingAs($super->user)->get($this->url);
+        $response->assertStatus(200);
+        $response->data('ignored')->assertNotContains($ticket1);
+        $response->data('ignored')->assertContains($ticket2);
+
+        $response = $this->actingAs($agent->user)->get($this->url);
+        $response->assertStatus(200);
```

Comment:
This status is the first clue that something's a bit off here. For a resource that a user shouldn't be able to access, a 403 (if we want them to know it exists) or a 404 (if we don't) would be more appropriate.